### PR TITLE
Always bootstrap rings for scaling support

### PR DESCRIPTION
**Pull request by ryshah**
Once https://github.com/blueboxgroup/swifttool/pull/8 is merged, swifttool would support modifying existing rings by adding/removing devices in current ring.

When the weights don't change, it will set weights to previous weights. However, this operation will not cause any data to reassign.

### CODE REVIEW

File: roles/swift-ring/tasks/main.yml [hunk 1]
```diff
@@ -1,15 +1,8 @@
...
-  shell: /usr/bin/test -f /etc/swift/account.ring.gz || /usr/bin/test -f /etc/swift/container.ring.gz || /usr/bin/test -f /etc/swift/object.ring.gz
-  failed_when: False
-  register: should_boostrap
-
 - name: drop our ring configuration
   copy: src={{ swift_ring.ring_definition_file }} owner=root group=root
         dest=/etc/swift/ring_definition.yml mode=644 backup=yes
-  when: not should_boostrap|success
 
 - name: setup swift rings
```
Comment (paulczar) [inline roles/swift-ring/tasks/main.yml]: should we set this to be a handler and to be notified by the "drop our ring configuration" task ?   That way it only runs when the ring config file changes...

Comment (ryshah) [inline roles/swift-ring/tasks/main.yml]: That is good idea! I will make the proposed changes...

**Comment by j2sol** [comment]
retest

**Comment by ryshah** [comment]
Dependent on https://github.com/blueboxgroup/swifttool/pull/8

**Comment by j2sol** [comment]
I like this change. But we can't merge it until swiftool PR gets merged.

**Comment by j2sol** [comment]
I'm just going to label it future to prevent me from merging this at the wrong time.

**Comment by ryshah** [comment]
@j2sol Can you please merge this one when you get chance? blueboxgroup/swifttool#8 has been merged

**Pull request merged**